pr-checks/checks/MilestoneCheck.ts

```diff
@@ -24,7 +24,17 @@ export class MilestoneCheck extends Check {
 			async (ctx) => {
 				const pr = context.payload.pull_request as EventPayloads.WebhookPayloadPullRequestPullRequest
 
-				if (pr && pr.milestone) {
+				if (!pr) {
+					return this.failure(ctx, '')
```

Comment:
I think this was originally bugged. If `pr` is not set, accessing `pr.head` would have caused a runtime error.